app/helpers/eventable.rb

```diff
@@ -12,9 +12,11 @@ def log_event(object:, organization: nil, type:)
       raise "No organization for event: #{type}" if organization.blank?
 
       EventJobs::CreateAndEmit.perform_async(
-        data: object.serialize,
-        organization: organization.id,
-        type: type
+        object.serialize,
+        object.class.name,
+        object.id,
```

Comment:
The interface for this is
`def perform(data, event_object_type, event_object_id, organization_id, type)`

Getting the object's class name and ID are the right things for the 2nd and 3rd params, right @ryanwjackson ?

Reply:
Yes, this is right.